### PR TITLE
[BugFix] start_fe.sh unary operator expected error

### PR DESCRIPTION
**Pull request by padmejin**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Fix a bug introduced by https://github.com/StarRocks/starrocks/pull/12432.
```
$ ./bin/start_fe.sh --daemon
./bin/start_fe.sh: line 90: [: -eq: unary operator expected
```
It's because the default of ENABLE_DEBUGGER is unset.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: bin/start_fe.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ eval set -- "$OPTS"
 RUN_DAEMON=0
 HELPER=
 HOST_TYPE=
-ENABLE_DEBUGGER=
+ENABLE_DEBUGGER=0
```
Comment (imay) [inline bin/start_fe.sh:36]: seems we can drop the echo in line 48 too

Comment (padmejin) [reply bin/start_fe.sh:36]: done

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**